src/molecules/formfields/util/BaseFieldGroup/index.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/require-render-return */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class BaseFieldGroup extends React.Component {
+  static propTypes = {
+    input: PropTypes.object,
+    meta: PropTypes.object,
+  }
+
+  componentWillMount() {
+    document.addEventListener('click', this.handleClickOutside, false);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.handleClickOutside, false);
+  }
+
+  wrapperReference = null;
+
+  clickIsLeaving = true;
```

Comment:
Can these be in the state of this component? Or will references to `this` be broke otherwise?

Reply:
i copied this from life-web and it has always worked so I'm just gunna keep as is